File: codility/TieRopes.js
function solution(K, A) {
    // Implement your solution here
    let sum = 0;
    let counter = 0;
    for (let i = 0; i < A.length; i++) {
        sum += A[i];
        if (sum >= K) {
            sum = 0;
            counter++;
        }
    }
    return counter;
}

File: codility/python/TieRopes.py
def solution(K, A):
    # Implement your solution here
    n = len(A)
    count = 0
    temp_sum = 0
    for i in range(n):
        a = A[i]
        if temp_sum + a >= K:
            count += 1
            temp_sum = 0
        else:
            temp_sum += a
    return count